### PR TITLE
[#4987] apps/budgeting/__tests__: add snapshot for budgeting list without items

### PR DESCRIPTION
**Pull request by phillimorland**
This is just testing when there are no items

### CODE REVIEW

File: meinberlin/apps/budgeting/assets/BudgetingProposalList.jsx [hunk 1]
```diff
@@ -56,6 +53,9 @@ export const BudgetingProposalList = (props) => {
                     onPaginate={newUrl => onPaginate(newUrl)}
                   />}
               </>
+              )
+            : (
+              <div>{django.gettext('Nothing to show')}</div>
```
Comment (phillimorland) [inline meinberlin/apps/budgeting/assets/BudgetingProposalList.jsx:58]: @khamui the snapshots always show this version, never the empty list version. I have tried a number of ways to change that but i'm still stumped, maybe we can have a look together?

File: meinberlin/apps/budgeting/assets/BudgetingProposalList.jsx [hunk 2]
```diff
@@ -48,6 +50,21 @@ export const BudgetingProposalList = (props) => {
               pageCount={meta.page_count}
               onPaginate={newUrl => onPaginate(newUrl)}
             />}
+        </>
+      )
+    } else {
+      list = (
+        <span>{django.gettext('Nothing to show')}</span>
```
Comment (goapunk) [inline meinberlin/apps/budgeting/assets/BudgetingProposalList.jsx:57]: Did you add this new empty state only for the test or was that something we wanted to have anyway?

Comment (phillimorland) [reply meinberlin/apps/budgeting/assets/BudgetingProposalList.jsx:57]: this was there in the django template but was forgotton in the jsx, adding test meant that I found this out (even if test isn't quite working)

**Comment by khamui** [comment]
cool

**Pull request merged**